ursa.ino

```diff
@@ -146,7 +146,13 @@ void loop() {  // on core 1. the balencing control loop will be here, with the g
     digitalWrite(LED_BUILTIN, HIGH);
 
     if (!wasRobotEnabled) {  // the robot wasn't enabled, but now it is, so this must be the first loop since it was enabled. re set up anything you might want to
-      // TODO: turn on stepper motors
+      // TODO: turn on stepper motors (complete?)
+      void setupMotors() { // function turns on stepper motors using an enable PIN on the drivers
+      pinMode(ENS_PIN GPIO_NUM_23, OUTPUT) // sets this PIN as an output
```

Comment:
One more thing, use just ENS_PIN to refer to the pin, it equals GPIO_NUM_23 which equals the number pin for the esp32 to change